boot/bootdata/hivesys.inf

```diff
@@ -1965,6 +1965,18 @@ HKLM,"SYSTEM\CurrentControlSet\Services\WlanSvc","ObjectName",0x00000000,"LocalS
 HKLM,"SYSTEM\CurrentControlSet\Services\WlanSvc","Start",0x00010001,0x00000002
 HKLM,"SYSTEM\CurrentControlSet\Services\WlanSvc","Type",0x00010001,0x00000110
 
+; W32TIME service
+HKLM,"SYSTEM\CurrentControlSet\Services\W32Time","DisplayName",0x00000000,%W32TIME_SERVICE%
```

Comment:
I think we'll want this to be a dll loaded in svchost instead of its own process. This means making w32time a dll and referencing svchost.exe here. You can see how it's done in e.g. shsvcs, srvsvc, wkssvc.

Reply:
I had looked at this and it seemed beyond by capabilities, but I will review again. Thanks.

Reply:
@ThFabba In fact I hate this svchost.exe mess. What's the point of running multiple services inside one process? Saving a few KB memory? "It's what Windows does"? I'd prefer each and every service would run with it's own process and executable.